example/main.tf

```
@@ -0,0 +1,51 @@
+module "vpc" {
+  source    = "git::https://github.com/cloudposse/terraform-aws-vpc.git?ref=master"
+  namespace = "example"
+  stage     = "dev"
+  name      = "app"
+}
+
+module "subnets" {
+  source             = "git::https://github.com/cloudposse/terraform-aws-dynamic-subnets.git?ref=master"
+  namespace          = "example"
+  stage              = "dev"
+  name               = "app"
+  region             = "us-west-2"
+  vpc_id             = "${module.vpc.vpc_id}"
+  igw_id             = "${module.vpc.igw_id}"
+  cidr_block         = "10.0.0.0/16"
+  availability_zones = ["us-west-2a", "us-west-2b"]
+}
+
+module "elasticsearch" {
+  source                         = "git::https://github.com/cloudposse/terraform-aws-elasticsearch.git?ref=tags/0.1.5"
```

```suggestion
  source                         = "git::https://github.com/cloudposse/terraform-aws-elasticsearch.git?ref=master"
```

use `master` for consistency

---

Sure, I was going for a known working release

---

it's kind of a chicken and the egg - also easily forgotten to update. We tend to pin our `README.md` examples to `master` so users who copy and paste get something that probably works.